### PR TITLE
refactor(billing services): update to standards

### PR DESCRIPTION
**Pull request by jniles**
This commit updates the billing services module in the following ways:
 1. It uses inline filtering like the other registries.
 2. Fixes a bug in rendering the account field - now only the number is
 displayed to avoid server API changes.
 3. Account dropdown filters out title accounts
 4. Sorting is enabled on the registry
 5. Use performance improvements for the ui-grid

### CODE REVIEW

File: client/src/modules/billing-services/billing-services.ctrl.js [hunk 1]
```diff
@@ -11,55 +12,63 @@ BillingServicesController.$inject = [
...
+    displayName: '',
+    enableSorting : false,
+    cellTemplate : actionTemplate,
+  }];
 
   // these options are for the ui-grid
   vm.options = {
     appScopeProvider: vm,
     enableSorting : true,
+    flatEntityAccess  : true,
```
Comment (sfount) [inline client/src/modules/billing-services/billing-services.ctrl.js:66]: :+1: incrementally making sure our older grids are using the latest standards

File: package.json [hunk 2]
```diff
@@ -56,7 +56,7 @@
     "express-handlebars": "^3.0.0",
     "express-session": "^1.15.4",
     "gulp-merge-json": "^1.1.0",
-    "helmet": "^3.8.0",
+    "helmet": "^3.8.1",
```
Comment (sfount) [inline package.json]: I don't think these version updates are related to this PR - could they be separated?

Comment (jniles) [reply package.json]: Cheers, I'll remove them.

**Comment by jniles** [comment]
@sfount, updates have landed!

File: client/src/modules/billing-services/billing-services.ctrl.js [hunk 3]
```diff
@@ -94,17 +102,17 @@ function BillingServicesController($state, BillingServices, Accounts, Notify, bh
...
 
     // retrieve a detailed list of the billing services in the application
     BillingServices.read(null, { detailed : 1 })
       .then(function (billingServices) {
 
         // make a pretty human readable account label
+        // TODO(@jniles) - make this use the account_label.  Requires changing
+        // the server
         billingServices.forEach(function (service) {
-          service.account = Accounts.label(service);
```
Comment (sfount) [inline client/src/modules/billing-services/billing-services.ctrl.js:107]: Is the `Accounts` library used anymore here? I think the import can be removed.

**Review by sfount — APPROVED** [review]
Changes LGTM. This is a great re-factor! I think you make one of the controller imports redundant in this PR so it would be good to remove that. This can be merged any time.

File: client/src/modules/billing-services/billing-services.ctrl.js (hunk 3 above)
Comment (jniles) [reply client/src/modules/billing-services/billing-services.ctrl.js:107]: Aha - I'll get right on that.

**Comment by jniles** [comment]
@sfount second round of changes have landed!

**Comment by jniles** [comment]
bors r=sfount

**Pull request merged**